### ISSUE TITLE
Graceful recovery from de-authorizing an account

### ISSUE THREAD

**Issue by fschaap** [bleakgrey/tootle#87]
Currently, when you de-authorize an account from being accessed by the token associated with Tootle, Tootle will just display an error message. There is no mechanism (that I could discover) to:

1. re-authorize the existing account, or
2. delete the account and add it again.

Yes, I could add the account again, but I could not delete the de-authorized account, so it remained in limbo stuck in the interface. I ended up deleting the `accounts.json` file to remove my de-authorized account.

**Comment by bleakgrey** [comment]
I don't think this is intended behavior. What error does it display?

**Comment by fschaap** [comment]
It briefly shows "Error: unauthorized" and then that message slides away.

**Comment by bleakgrey** [comment]
Ah, you mean the account token was revoked earlier?

**Comment by fschaap** [comment]
Yes :-)

**Comment by bleakgrey** [comment]
Currently the supposed way to "re-authorize" an account is to remove it and re-add it again in accounts menu. I can't quite understand why its removing fails though, it's literally just writing to a file.

I'll try to reproduce this with a revoked token and see what I can do there

**Comment by fschaap** [comment]
Ah, I think I see what is going wrong. Because [the icons are not showing correctly](https://github.com/bleakgrey/tootle/issues/20) I did not see there was a remove/delete icon in the accounts menu. For it me it just shows a missing-icon icon. This icon also does not have a hover-text, so I simply did not see any option in the app to remove accounts. I just clicked the icon and the account was deleted. So the functionality works!

This ticket can be closed then :-)
